sonar-lits-plugin/src/main/java/com/sonarsource/lits/IssuesChecker.java

```diff
@@ -148,14 +152,16 @@ private static void forceDelete(File file) {
     }
   }
 
-  void save() {
+  void save(FileSystem fileSystem) {
     forceDelete(newDumpFile);
     List<String> messages = new ArrayList<>();
     MessageException exception = null;
     if (different) {
       LOG.info("Saving " + newDumpFile);
       Dump.save(dump, newDumpFile);
       messages.add("Issues differences: " + differences);
+      new HtmlReport(fileSystem).print(new LitsReport(differentIssues, fileSystem));
```

Comment:
There are costs to generate report, so maybe it should be optional for the users of this plugin who rely on SonarQube UI and do not need this report.